test/integration/security/unauthorized-requests.js

```diff
@@ -0,0 +1,339 @@
+describe(require('../../__fixtures/utils/test_helper').create().testName(__filename, 3), function () {
+
+  var path = require('path');
+  var expect = require('expect.js');
+  var happn = require('../../../lib/index');
+  var service = happn.service;
+
+  function doRequest(path, token, query, callback, excludeToken) {
```

Comment:
for readability, rather follow callback-last convention (even if it necessitates always specifying excludeToken)

Reply:
done